concepticondata/concepticon.tsv

```diff
@@ -2671,3 +2671,4 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY
 2670	ACROSS	Spatial relations	Located directly across from something else, or from each other.	Other
 2671	PENIS SHEATH	Clothing and grooming	Sheath to cover a penis	Person/Thing
 2672	HEADWATERS	The physical world	The source of a river or stream	Person/Thing
+2673	BIRD OF PARADISE	Animals	A bird of the family Paradisaeidae.	Person/Thing
```

Comment:
Yes, I think the bigger danger (for lexibank etc) comes from indiscriminately merging things like "Bird (of specific species)" with "Bird (generic term)". I thought about leaving "bush hen" unmapped, but left it as "hen" (disagree?)